module/VuFind/src/VuFind/View/Helper/Root/IconFactory.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * Captcha helper factory.
```

Comment:
Copied and pasted comment.

Reply:
(Yes, it is true that I copied and pasted my comment about your copied and pasted comments in order to comment on them twice).